### PR TITLE
Upgrade to Capacitor v6

### PR DESCRIPTION
**Pull request by RickJeroen**
See: https://capacitorjs.com/docs/updating/plugins/6-0

### CODE REVIEW

**Comment by marshall86** [comment]
when are you going to complete the migration?

**Comment by RickJeroen** [comment]
This PR is ready to merge

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@capacitor-community/intercom",
-  "version": "5.0.1",
+  "version": "6.0.0",
```
Comment (stewones) [inline package.json:3]: can you revert it to 5.0.1? we have a script which auto bumps this

**Comment by stewones** [comment]
Thanks @RickJeroen LGTM

**Pull request merged**